Tools/buildbot/remoteDeploy.bat

```diff
@@ -2,6 +2,7 @@
 rem Used by the buildbot "remotedeploy" step.
 setlocal
 
+set PATH=%PATH%;%SystemRoot%\SysNative\OpenSSH
```

Comment:
Unfortunately, `SysNative` only works in 32-bit command prompt. Add both `SysNative` and `System32` here to ensure it works even if we switch to a 64-bit process one day.

Reply:
System32\OpenSSH is always on the path right now, whether I run C:\Windows\SysWOW64\cmd.exe or C:\Windows\System32\cmd.exe.  I can add it again, but it seemed redundant